api/pkg/apis/v1alpha1/providers/stage/create/create.go

```diff
@@ -140,7 +140,7 @@ func (i *CreateStageProvider) Process(ctx context.Context, mgrContext contexts.M
 	lastSummaryMessage := ""
 	switch objectType {
 	case "instance":
-		objectNamespace := stage.ReadInputString(inputs, "objectNamespace")
```

Comment:
objectNamespace specifies the namespace where the object is to be created. This is different from "__namespace", which is a "system property" to carry namespace information of the trigger data.

Reply:
Oh, I see. objectNamespace is used to support creating/materializing objects to a different namespace from where the activation is triggered.